### ISSUE TITLE
ignoring subdirectory

### ISSUE THREAD

**Issue by raddimm** [humy2833/FTP-Simple#151]
Hi, i have a problem ... 

I use command remote directory open to workspace but i cannot open subfolder correctly
There is only: [DIR]something

Anyway to open folders and subfolders?

**Comment by PROX1239** [comment]
Me too have this problem. We need help.

**Comment by skopz356** [comment]
Yes, I have this problem too .. I am using Vs Code on Windows 10.

**Comment by leonidaswander** [comment]
solution to the problem?

**Comment by jorgehdev** [comment]
> solution to the problem?

I didn't find anything. You would use ftp-kr

**Comment by bmpf** [comment]
Same here.
I thought it was a conflict with other packages, I uninstalled everything and the problem is still there.

**Comment by bmpf** [comment]
Rollback to the version v1.52 https://code.visualstudio.com/updates/v1_52#_source-control , its working.

**Comment by bmpf** [comment]
version v1.53 & version v1.54 not working.
v1.52 is working fine.
